src/main/java/appeng/util/Platform.java

```diff
@@ -571,11 +569,11 @@ public static ItemStack findMatchingRecipeOutput( final InventoryCrafting ic, fi
 	}
 
 	@SideOnly( Side.CLIENT )
-	public static List getTooltip( final Object o )
+	public static List<String> getTooltip( final Object o )
 	{
 		if( o == null )
 		{
-			return new ArrayList();
+			return new ArrayList<>();
 		}
 
 		ItemStack itemStack = ItemStack.EMPTY;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "itemStack". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1854)